packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx

```diff
@@ -44,11 +35,10 @@ export const RemoteScreenShare = React.memo(
 
     useEffect(() => {
       return () => {
-        if (isVideoStreamOn) {
-          onDisposeRemoteStreamView && onDisposeRemoteStreamView(userId);
-        }
+        // TODO: Isolate disposing behaviors for screenShare and videoStream
```

Comment:
I don't understand this change.
What would happen when this component tries to call onDisposeRemoteStreamView but there is no remoteVideoStream?

Reply:
It's totally fine, you can call it hundred times, and it is just doing a check in the map object